src/components/input/Input.tsx

```diff
@@ -0,0 +1,53 @@
+import { useRef, useState } from 'react';
+
+type InputPropsType = {
+  inputName?: string;
+  inputType?: string;
+  placeholder?: string;
+};
```

Comment:
`input`에 대한 타입을 받으면 좋을 것 같다는 생각을 했습니다..!

```suggestion
type InputPropsType = {
  inputName?: string;
} & React.InputHTMLAttributes<HTMLInputElement>;

...

<input
  {...props}
  ref={inputRef}
  value={inputValue}
  onChange={(e) => setInputValue(e.target.value)}
/>
```